net/api/chunked_demo.cc

```diff
@@ -109,9 +109,39 @@ using namespace bricks::net::api;
 using namespace bricks::cerealize;
 using bricks::time::Now;
 using bricks::strings::Printf;
+using bricks::net::HTTPResponseCode;
 
 DEFINE_int32(port, 8181, "The port to serve chunked response on.");
 
+struct LayoutCell {
+  std::string meta_url = "/meta";
+  
+  template <typename A>
+  void serialize(A& ar) {
+    ar(CEREAL_NVP(meta_url));
+  }
+};
+
+struct LayoutItem {
+  std::vector<LayoutItem> row;
+  std::vector<LayoutItem> col;
+  LayoutCell cell;
+
+  template <typename A>
+  void serialize(A& ar) {
```

Comment:
Thanks. It's for serialization only; otherwise we'll have to change the layout definition structure. Please help me to write this assert expression, I'm not that confident in C++.